web/themes/custom/pineapple/sass/layouts/views/_view--recipes.scss

```diff
@@ -0,0 +1,101 @@
+.view-recipes {
+
+  form {
+    padding: 0;
+
+    .form--inline {
+      width:100%;
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between;
+      align-items: flex-start;
+      margin-bottom: $unit *3;
```

Comment:
If you're gonna add this margin bottom it should be on the form.

Reply:
Done.